destinationCreation.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+      <meta charset="UTF-8"/>
+      <title>Destination Creation</title>
+      <link rel="stylesheet" href="destinationCreation.css"/>
+      <script src="destinationCreation.js"></script>
+    </head>
+    <header>
+    </header>
+    <body>
+      <div id="content">
+        <h1>Destination Creation Form</h1>
+        <div id="form-container">
+          <form action="/data" method="post">
+            <label for="name-input">Enter the name of the destination: *</label>
+            <textarea name="name" id="name-input" required></textarea>
+            <p>Enter the latitude and longitude of the destination:</p>
+            <label for="lat-input">Latitude: *</label>
+            <textarea name="latitude" id="lat-input" required></textarea>
+            <label for="long-input">Longitude: *</label>
+            <textarea name="longitude" id="long-input" required></textarea>
```

Comment:
I recomment `lng` rather than `long` as the abbreviation. `long` may lead to confusion with the `long` integer type.